homepage.css

```diff
@@ -144,54 +155,212 @@ ul li:hover ul{
 	height:214px;
 	background-color:white;
 	margin-top:50px;
+	position: relative;
+    width: 100%;
+}
+
+.images1 img{
+	height:214px;
+	margin-left:25px;
 }
 
 .images2{
 	height:214px;
 	background-color:white;
 	margin-top:50px;
+	position: relative;
+    width: 100%;
+}
+
+.images2 img{
+	height:214px;
+	margin-left:25px;	
 }
 
 .images3{
 	height:214px;
 	background-color:white;
 	margin-top:50px;
+	position: relative;
+    width: 100%;
+}
+
+.images3 img{
+	height:214px;
+	margin-left:25px;
 }
 
 .images4{
 	height:214px;
 	background-color:white;
 	margin-top:50px;
-
+	position: relative;
+    width: 100%;
 }
 
+.images4 img{
+	height:214px;
+	margin-left:25px;
+}
 
 .images5{
 	height:214px;
 	background-color:white;
 	margin-top:50px;
+	position: relative;
+    width: 100%;
+}
+
+.images5 img{
+	height:214px;
+	margin-left:25px;
 }
 
 .images6{
 	height:214px;
 	background-color:white;
 	margin-top:50px;
+	position: relative;
+    width: 100%;
+}
+
+.images6 img{
+	height:214px;
+	margin-left:25px;
 }
 
 .images7{
 	height:214px;
 	background-color:white;
 	margin-top:50px;
+	position: relative;
+    width: 100%;
+}
+
+.images7 img{
+	height:214px;
+	margin-left:25px;
 }
 
 .images8{
 	height:214px;
 	background-color:white;
 	margin-top:50px;
-	
+	position: relative;
+    width: 100%;
 }
 
-.content_4 img{
+.images8 img{
+	height:214px;
 	margin-left:25px;
-	border: 2px solid black;
+}
+
+
+
+.image1 {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+.image2 {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+.image3 {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+.image4 {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+.image5 {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+.image6 {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+.image7 {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+.image8 {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+
+.overlay {
```

Comment:
31/08/2021: Set up the overlay techniques. When hover on it will display the background in that div box and display a link that contain that link that that div box con contain: Eg div box images 1 when hover will display a link called Indie and when click on it will deliver to a page that contain all the song that in the genre Indie.